scripts/metadata_comparison/metadata_comparison/lib/operations_digesters.py

```diff
@@ -163,11 +230,11 @@ def localization_time_seconds(self) -> float:
         return (events[-1] - events[0]).total_seconds()
 
     def user_command_time_seconds(self) -> float:
-        started_running_description = "^Started running .* /cromwell_root/script\"$"
+        started_running_description = "^Started running .*/cromwell_root/script\"$"
         started_running_events = [dateutil.parser.parse(d.get('timestamp')) for d in
                                   self.event_with_description_like(started_running_description)]
 
-        stopped_running_description = "^Stopped running \"/bin/bash /cromwell_root/script\""
```

Comment:
Good catch! I wonder whether we ought to be running these scripts more often to catch awkward things like this?

Reply:
Wasn't that hard to catch, this completely broke the digester. I was wondering the same thing myself but I'm not sure how much longer the current set of scripts is going to see heavy use. Sure we'd like to monitor our performance on PAPI over long periods of time but that's just one of a million nice things we might want to work on but don't have time for...

Reply:
I guess if we ever do long-term monitoring of performance and speed, then we'd get regression testing of things like this built-in